src/chatlog/chatlinecontentproxy.h

```diff
@@ -15,9 +15,12 @@ class ChatLineContentProxy : public ChatLineContent
     Q_OBJECT
 
 public:
+    /**
+     * @brief Type tag to avoid dynamic_cast of contained QWidget*
+     */
     enum ChatLineContentProxyType
     {
-        GenericType,
+        GenericType = 0,
```

Comment:
Should GenericType and FileTransferWidgetType really have the same value? If so, this enum feels a bit silly to keep around?

Reply:
Added a TODO to figure that out. I don't know what this enum is supposed to do. There's not much use of it except in an assert that will always pass by construction.